1-index.html

```diff
@@ -44,13 +44,12 @@ <h2>Help needed?</h2>
     </p>
   </div>
   <div class="fx-col-100-xs fx-col-50-l">
-    <h2>Jamulus is Open Source</h2>
-    <p>
-      Ideas? Found a bug? Want to contribute? Since Jamulus is open source you can help us! Find the project on <a href="https://sourceforge.net/projects/llcon/" target="_blank" rel="noreferrer">SourceForge</a> and the <a
-        href="https://github.com/corrados/jamulus" target="_blank" rel="noreferrer">source code on GitHub</a>!
-      Jamulus is licensed under the (<a href="http://www.gnu.org/licenses/gpl-2.0.html" target="_blank" rel="noreferrer">GPL,
-        GNU General Public License</a>).
-    </p>
+      <h2>Jamulus is a <a href="https://www.gnu.org/philosophy/free-sw.en.html" target="_blank" rel="noreferrer" title="What is free software?">free software</a></h2>
```

Comment:
Are you sure about "is **a** free software"? Shouldn't it be "is free software"?

Reply:
> Are you sure about "is **a** free software"? Shouldn't it be "is free software"?

No, not sure in english.
In French the cardinal determinant is required.
We would not write "Jamulus est logiciel libre" but "Jamulus est *un* logiciel libre" because "software" is not an adjective.

If I check the French to English translation with Deepl, both seem possible in English. But the default is without **a**.

I can modify with : _Jamulus is free and open source software_ if you think is better using FOSS instead free software only.

Reply:
> I can modify with : Jamulus is free and open source software if you think is better using FOSS instead free software only.

The advantage of using "FOSS" is that there is no longer any ambiguity with the word "free".

Reply:
BTW: please rename the file to 1-fr-index.html

Reply:
> Another alternative I've seen several times is to use "libre software".

To choose from, I prefer "free and open source software". ;)
I'll change it like this.

Reply:
> To choose from, I prefer "free and open source software". ;)
> I'll change it like this.

@trebmuh for the french version I think we can use « logiciel libre » it is sufficiently meaningful and understandable.
« logiciel libre à sources ouvertes » or an equivalent formulation is never used (I have never seen).

Reply:
> @trebmuh for the french version I think we can use « logiciel libre » it is sufficiently meaningful and understandable.
« logiciel libre à sources ouvertes » or an equivalent formulation is never used (I have never seen).

Agreed.